app/src/main/java/com/example/todoapp/ui/screens/homescreen/HomeScreen.kt

```diff
@@ -122,77 +134,85 @@ fun NotesList() {
       .background(color = Color.White, shape = RoundedCornerShape(size = 10.dp)),
     horizontalAlignment = Alignment.CenterHorizontally
   ) {
-    AnimatedVisibility( isNotesVisible) {
-      Column (horizontalAlignment = Alignment.CenterHorizontally){
-      MyTextField(
-        label = stringResource(R.string.title),
-        placeholder = stringResource(R.string.note_title),
-        modifier = Modifier.padding(horizontal = 10.dp),
-        onValueChanged = {}
-      )
-      MyTextField(
-        label = stringResource(R.string.description),
-        placeholder = stringResource(R.string.note_description),
-        modifier = Modifier.padding(horizontal = 10.dp),
-        onValueChanged = {}
-      )
+    AnimatedVisibility(isNotesVisible) {
+      Column(horizontalAlignment = Alignment.CenterHorizontally) {
+        MyTextField(
+          label = stringResource(R.string.title),
+          placeholder = stringResource(R.string.note_title),
+          modifier = Modifier.padding(horizontal = 10.dp),
+          onValueChanged = {
+            onTitleChanged(it)
+          }
```

Comment:
Just for your info, you can ignore this or not but you can do it this way,  kotlin supports functional programming so functions can be passed as parameters/variables 😉
```suggestion
          onValueChanged = onTitleChanged
```